app/models/project.rb

```diff
@@ -22,6 +23,10 @@ class Project < ApplicationRecord
   scope :public_and_not_forked,
         -> { where(project_access_type: "Public", forked_project_id: nil) }
 
+  scope :public_access, -> { where(project_access_type: "Public") }
+
+  scope :author, ->(author_id) { where(author_id: author_id) }
```

Comment:
```suggestion
  scope :by, ->(author_id) { where(author_id: author_id) }
```

```ruby
 Project.author(author_id)
```

```ruby
 Project.by(author_id)
```
Second one seems express the context better